### PR TITLE
Info page

### PR DESCRIPTION
**Pull request by crisgarner**
Added dates in tallying and distribution

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export const config = {
   registrationEndsAt: new Date(process.env.NEXT_PUBLIC_REGISTRATION_END_DATE!),
   reviewEndsAt: new Date(process.env.NEXT_PUBLIC_REVIEW_END_DATE!),
   resultsAt: new Date(process.env.NEXT_PUBLIC_RESULTS_DATE!),
+  votingEndsAt: new Date(process.env.NEXT_PUBLIC_VOTING_END_DATE!),
```
Comment (kittybest) [inline src/config.ts:20]: This is removed by me from another PR: #124 
Since we fetch the ending date from the contract

**Comment by ctrlc03** [comment]
@kittybest will this be superseded by the new design? should we close? @crisgarner

**Comment by crisgarner** [comment]
This will change with the new design, I think we can close it

**Comment by kittybest** [comment]
close it by implementing new design already.

**Pull request closed without merging**